### DISCUSSION TITLE
import songs into container station

### DISCUSSION THREAD

**Discussion by MXlt853** [beetbox/beets#4466] [Q&A] [answered]
I'm running beets in docker compose in the container station I'm trying to import the files into the folder I configured but the message that the directory or files doesn't exist always appears

**Comment by arsaboo** [comment, marked as answer]
You need to use the path you have configured in the docker container. So, if you have configured `-v </path/to/music/library>:/music` in your docker, you will use `beet import /music` inside the docker terminal.
